forms-web-app/src/lib/documents-api-wrapper.js

```diff
@@ -0,0 +1,54 @@
+const fetch = require('node-fetch');
+const FormData = require('form-data');
+const uuid = require('uuid');
+
+const config = require('../config');
+const parentLogger = require('./logger');
+
+exports.createDocument = async (appeal, formData) => {
+  const path = `/api/v1/${appeal.id}`;
+
+  const correlationId = uuid.v4();
+  const url = `${config.documents.url}${path}`;
+
+  const logger = parentLogger.child({
+    correlationId,
+    service: 'Document Service API',
+  });
+
+  let apiResponse;
+  try {
+    const fd = new FormData();
+    fd.append('file', Buffer.from(formData.data));
+
+    apiResponse = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'multipart/form-data',
+        'X-Correlation-ID': correlationId,
+      },
+      body: fd,
+    });
+  } catch (e) {
+    logger.error(e);
+    throw new Error(e.toString());
+  }
+
+  if (!apiResponse.ok) {
+    logger.debug(apiResponse, 'Documents API Response not OK');
+
+    throw new Error(apiResponse.statusText);
+  }
+
+  const ok = (await apiResponse.status) === 202;
+
+  if (!ok) {
+    throw new Error(apiResponse.statusText);
+  }
+
+  return {
+    id: uuid.v4(),
+    applicationId: appeal.id,
+    name: 'tmp-2-1607684291243',
```

Comment:
DNM with this in place.

Reply:
If this is already in `master`, this will be preserved when this is rebased. I'll rebase from `master` just to be sure though